heartbeat/docs/heartbeat-options.asciidoc

```diff
@@ -5,17 +5,22 @@
 <titleabbrev>Set up Heartbeat monitors</titleabbrev>
 ++++
 
-The `heartbeat.monitors` section of the +heartbeat.yml+ config file specifies
-the list of `monitors` that Heartbeat uses to check your remote hosts to
-determine if they are available. Each `monitor` item begins with a dash (-) and
-specifies the type of monitor to use, the hosts to check, and other settings
-that control Heartbeat behavior.
+Heartbeat is configured by defining a set of `monitors` to check your remote hosts.
+These can be specified either inside the +heartbeat.yml+ config file, or in external
```

Comment:
I want to tighten up the language used here, but rather than entering comments on this PR, it'll be more efficient for me to come back later and edit these additions in a separate PR. So for now, I'll just point out stuff that is potentially confusing, rather than style issues. Created an issue here: https://github.com/elastic/beats/issues/8332